project.clj

```diff
@@ -27,7 +27,7 @@
 
   :min-lein-version "2.9.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "4.6.10"]
+  :parent-project {:coords [puppetlabs/clj-parent "4.6.11"]
```

Comment:
This update will be automatic when clj-parent is released. However, we will need to do the bump by hand in pe-pse.

Reply:
👍 

PR is at https://github.com/puppetlabs/pe-puppet-server-extensions/pull/1262 

I'll update this PR once clj-parent ships and after this repo is auto-updated